3.py

```diff
@@ -0,0 +1,22 @@
+"""
+Write a program to create "BCA.txt" file which contain information about BCA course. 
+count and print the total number of lines starting with ‘A’, ‘B’, and ‘C’ in "intro.txt" file.
+"""

```

Comment:
there is no import statement. is it ok or give error?

Reply:
It is ok.it will not give error because it is definition of the program.And (""") triple quote is used to comment in program